README.md

```diff
@@ -1,10 +1,187 @@
+- [**Getting started**](#getting-started)
+  - [Installation](#installation)
+    - [_Clone Project_](#clone-project)
+    - [_Install Project Dependencies_](#install-project-dependencies)
+  - [Environment Variables](#environment-variables)
+    - [_REACT_APP_BACKEND_HOST_](#react-app-backend-host)
+    - [_REACT_APP_ES_API_](#react-app-es-api)
+    - [_REACT_APP_ES_MULT_API_](#react-app-es-mult_api)
+    - [_REACT_APP_AUTH_](#react-app-auth)
+    - [_REACT_APP_CONFIGURATION_API_](#react-app-configuration-api)
+    - [_REACT_APP_EXPERIENCES_](#react-app-experiences)
+    - [_REACT_APP_ADD_COURSE_TO_LISTS_](#react-app-add-course-to-lists)
+    - [_REACT_APP_INTEREST_LISTS_ALL_](#react-app-interest-lists-all)
+    - [_REACT_APP_USER_SUBSCRIPTION_LISTS_](#react-app-user-subscription-lists)
+    - [_REACT_APP_USER_INTEREST_LISTS_](#react-app-user-interest-lists)
+  - [Creating a local env](#creating-a-local-environment-file)
+    - [_Walk through_](#walk-through)
+    - [_Template_](#env-template)
+  - [Important Notes](#important-notes)
+- [**Available Scripts**](#available-scripts)
+  - [yarn start](#yarn-start)
+  - [yarn test](#yarn-test)
+  - [yarn build](#yarn-build)
+  - [yarn coverage](#yarn-coverage)
+
+---
+
+## Installation
+
+### Clone Project
+
+```bash
+git clone git@github.com:OpenLXP/openlxp-xms-ui.git
+```
+
+### Install Project Dependencies
+
+Start off by verifying that you have `yarn` installed.
+
+```ps1
+yarn -version
+```
+
+If `yarn` is not installed use the following command to install it. This will install the `yarn` package manager globally on your system.
+
+_**global install**_
+
+```ps1
+npm install yarn -g
+```
+
+_**local install**_
+
+```ps1
+npm install yarn
+```
+
+Once yarn has been installed you will need to install the project dependencies. Using the following command we will manually set your yarn version for this project.
+
+```ps1
+yarn set version 1.22.1
+```
+
+After the version has been installed and set we will install the dependencies. Using the following command we will install all the project dependencies.
+
+```bash
+yarn install package.json
+```
+
+---
+
+## Getting started
+
+### Environment Variables
+
+This project makes use of globally available environment variables. Below are the required environment variables required for this project.
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
```

Comment:
Needs to be updated with the relevant info. The vars described should match the .env